src/BuildIntegration/BuildReadyToRunMultiModule.proj

```diff
@@ -0,0 +1,40 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <IlcCompileDependsOn>ComputeIlcCompileInputs;BuildOneReadyToRunMultiModule;AddModuleDependencies</IlcCompileDependsOn>
+    <IlcMultiModule>true</IlcMultiModule>
```

Comment:
We should not be setting `IlcMultiModule` to true. Please remove this and make sure the tests still run. Delete the previous run's test binaries, too, before re-running just to be sure things aren't accidentally still working (bugs in build system changes can be easily masked by old build binaries)..